org.lflang/src/org/lflang/generator/ReactionInstanceGraph.xtend

```diff
@@ -134,7 +133,7 @@ class ReactionInstanceGraph extends DirectedGraph<ReactionInstance> {
      * @return true if the assignment was successful, false if it was not, 
      * meaning the graph has at least one cycle in it.
      */
-    protected def DirectedGraph<ReactionInstance> assignLevels() {
+    private def DirectedGraph<ReactionInstance> assignLevels() {
```

Comment:
The return comment for this method is now incorrect. Also the sentence before the return part.